tests/foreign_libc.txt

```diff
@@ -0,0 +1,8 @@
+:argc 0
+:stdin 0
+
+:stdout 0
+
```

Comment:
run_tests.py not capturing output?

Reply:
Need to call fflush for C programs because they interact differently with a PIPE (as opposed to a terminal)